pkgs/development/interpreters/elixir/generic-builder.nix

```diff
@@ -0,0 +1,77 @@
+{ pkgs, stdenv, fetchFromGitHub, erlang, rebar, makeWrapper,
+  coreutils, curl, bash, debugInfo ? false }:
+
+{ baseName ? "elixir"
+, version
+, minimumOTPVersion
+, sha256 ? null
+, rev ? "v${version}"
+, src ? fetchFromGitHub { inherit rev sha256; owner = "elixir-lang"; repo = "elixir"; }
+}:
+
+let
+  inherit (stdenv.lib) getVersion versionAtLeast;
+
+in
+  assert versionAtLeast (getVersion erlang) minimumOTPVersion;
```

Comment:
`stdenv.lib.getVersion` does not work at least for erlangR16 (I don't remember about other versions). That's the reason I have introduced erlang-specific version of this function here:
https://github.com/NixOS/nixpkgs/blob/aba574c4faeed0a359324d9fb89c7aeffb08314d/pkgs/development/beam-modules/lib.nix#L15

Can you double check it works well for all Erlang versions present in Nixpkgs?